python/ray/_private/runtime_env/validation_test.py

```diff
@@ -0,0 +1,36 @@
+from python.ray._private.runtime_env import validation
+
```

Comment:
We already have `test_runtime_env_validation.py`, could you combine them?

Reply:
No plan at the moment, but I could put them in the same folder, if you want.

Let me explain my thoughts:
- They serve for different purpose. They shouldn't be placed in the same test suite. At the moment, we only have **integration test** for python features, which relies on the latest current version of `ray`' while I'm writing **unit test**, whose test target is single function or class.
- They have different runtime and config. Integration test generally requires external access, while unit test is and requires hermetic and self-contained (`bazel test` put tests in a sandbox to prevent all external accesses). They usually have different runtime as well, unit tests are small and quick (bazel by default limits a test for 300 sec).

Reply:
If you check `TestValidatePip` it's unit test not integration test.

Reply:
If you're fine about it, may I move all unit test into a separate unit test file?

Reply:
Yes. Having all unit tests in one place.

Reply:
Sounds good! I move a few test cases into my unit test file, also moved it under `tests` folder for aggregation.